assets/scripts/embedding_tests_dev_0.0.2.min.html

```diff
@@ -6,31 +6,63 @@ <h2>v0.0.2 (minified)</h2>
   the Oppia iframe to the containing page.
```

Comment:
Comments from the other file apply to this one too.

Reply:
done.